model/resource/host.yaml

```diff
@@ -79,6 +79,15 @@ groups:
           IPv4 Addresses MUST be specified in dotted-quad notation. IPv6 addresses
           MUST be specified in the [RFC 5952](https://www.rfc-editor.org/rfc/rfc5952.html) format.
         examples: ["192.168.1.140", "fe80::abc2:4a28:737a:609e"]
+      - id: mac
```

Comment:
I think this is the same controversy as `host.ip`.
We are adding `host.mac` which represends multiple mac addresses to be consistent with ECS and we again violated the otel recommendations.

Reply:
For reference: https://github.com/open-telemetry/semantic-conventions/pull/333/files

Reply:
@frzifus what would you suggest doing here to move forward?

Reply:
@AlexanderWert I assume you reference to:
>   - Do not modify an ECS name for the only reason of complying with [OpenTelemetry general attribute naming guidelines](https://opentelemetry.io/docs/specs/otel/common/attribute-naming/).

I am fundamentally unsure whether this guideline is a good idea. However, I don't have a very strong opinion on it, as I am not very familiar with ECS.

That said, if everyone agrees, I don't want to block the change.

cc @mx-psi